tests/integration_tests/process_input_tests.sh

```diff
@@ -35,6 +35,7 @@ test_invalid_file2_input(){
 }
 
 test_no_input_file(){
+    input_paths="."
```

Comment:
if you are testing no input file, then the input path should be empty as in: input_paths=""
If you pass an empty input_path, test will fail which is a valid failure because of the change in line 23 of the entrypoint.sh